api/swagger.yaml

```diff
@@ -3333,6 +3333,18 @@ definitions:
       Driver:
         description: "Name of the secrets driver used to fetch the secret's value from an external secret store"
         $ref: "#/definitions/Driver"
+      Templating:
+        description: "Templating driver, if applicable"
+        type: "object"
```

Comment:
Like `Driver` above, this is a `Driver` type (also perhaps take some of the description from the `ConfigSpec` struct), something like;

```yaml
Templating:
  description: |
    Templating driver, if applicable.

    Templating controls whether and how to evaluate the config payload as a
    template. If no driver is set, no templating is used.
  $ref: "#/definitions/Driver"
```